app/controllers/events_controller.rb

```diff
@@ -39,15 +42,16 @@ def get_my_events
     @my_registrations = []
     @my_events        = []
 
-    user_events = UserEvent.where(user_id: current_user.id, event_id: current_events)
-    user_events.each do |user_event|
-      @my_registrations.push user_event
-      @my_events.push user_event.event
+    unless current_user.nil?
```

Comment:
I presume this guards against unauthroized users. Is something like this needed for the current_user.admin? check above or does ruby magic make that work for unauthorized? (Like, current_user is never null, but can have the property nil?)

Reply:
@panpear Yeah, this is making sure the code block, which follows, and requires a `current_user` does not blow up.

So, we're not going to load a user's data if the user is not logged in.,